filters/include/pcl/filters/functor_filter.h

```diff
@@ -76,22 +77,48 @@ class FunctorFilter : public FilterIndices<PointT> {
    */
   void
   applyFilter(Indices& indices) override
+  {
+    const auto exec = executor::best_fit{};
+    applyFilter(exec, indices);
```

Comment:
```suggestion
    applyFilter(std::move(exec), indices);
```
Since we don't need this anymore

Reply:
Or go directly for
```suggestion
    applyFilter(executor::best_fit{}, indices);
```

Reply:
On hold since with the new best fit in v0.3 of the executor will not require this. Will update the method after v0.3 is integrated.

Reply:
Integrate new API